wpilibc/src/main/native/cpp/simulation/DCMotorSim.cpp

```diff
@@ -16,6 +16,7 @@ DCMotorSim::DCMotorSim(const LinearSystem<2, 1, 2>& plant,
                        const std::array<double, 2>& measurementStdDevs)
     : LinearSystemSim<2, 1, 2>(plant, measurementStdDevs),
       m_gearbox(gearbox),
+
```

Comment:
Please remove this newline.

Reply:
How do I add the new target branch to my fork.  It doesn't exist on my end to rebase from

Reply:
I get an error:  

fatal:  invalid upstream 'development'